controllers/favorites.js

```diff
@@ -0,0 +1,66 @@
+var express = require('express');
+var app = express();
+var router = express.Router();
+var db = require('../models');
+
+app.set('view engine', 'ejs');
+
+router.get("/", function(req, res) {
```

Comment:
Make sure you include tags in this route, so that you can show them for each favorite:
```
router.get("/", function(req, res) {
	db.favorite.findAll({
		include: [db.tag]
	}).then(function(films) {
		res.render('favorites/index.ejs', {films: films})
	});
});
```